Makefile

```diff
@@ -205,3 +200,21 @@ restart-ui: rm-issuer-imgs up run run-ui
 
 .PHONY: restart-ui-arm
 restart-ui-arm: rm-issuer-imgs up run-arm run-ui-arm
+
+
+# usage: make new_password=xxx change-this-default-password
+.PHONY: change-this-default-password
+change-this-default-password:
```

Comment:
Not sure if a user will understand what password is changing here. Maybe "change-vault-password" could make easier to understand?